product-releases/2021-03-04-multiple-standards.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: Multiple Standards
+author: Tyler Tsunekawa
+author_title: Software Engineer
+author_url: https://github.com/ttsunekawa
+author_image_url: https://assets-global.website-files.com/5fd1d1ab54430e3590726397/5fd1d1ab54430e379c7264e2_tyler-headshot.png
+tags: [standards]
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+Standards help measure and ensure service quality across multiple aspects of your infrastructure.
+
+<!--truncate-->
+
+## Team based standards? Organization based standards? You decide.
+
+Achieve production readiness and service quality across all services and maintained by the leaders of your organization.
+
+Multiple standards gives you the ability to set high level standards that everyone must follow while still giving your team leaders flexibility to set more granular rules also.
+
+<img
+  alt='Standards List'
+  src={useBaseUrl('img/multiple-standards/standards-page.png')}
+/>
+
+## See how your services are performing at all angles.
+
+With multiple standards, we give you the ability to view a service’s health in multiple ways.
+
+You can see how services are performing against a single standard, or see how a service is performing against all of it’s standards.
+
+These different views help give actionable items to all members of your team to help raise the bar of your organization.
+
+## Set requirements
+
+Help your organization work on what is important and decide which criteria are **Required** and which are **Recommended**.
+
+By prioritizing the work to be done, your organization’s services will reach production-readiness faster.
```

Comment:
maybe here it's good to have a picture of the status badges on the standards show page?